cloud/deploys/bin/azure/lib.sh

```diff
@@ -71,8 +71,79 @@ function azure::add_generated_secrets() {
   for key in "$@";
   do
     echo "Generating secret: ${key}"
-    local SECRET_VALUE="$(head /dev/urandom | LC_CTYPE=C tr -dc "${CHARSET}" | cut -c -40)"
+    local SECRET_VALUE="$(head /dev/urandom | LC_CTYPE=C tr -dc "${CHARSET}" |LC_CTYPE=C cut -c -40)"
     echo "Setting secret: ${key}"
     azure::add_secret "${VAULT_NAME}" "${key}" "${SECRET_VALUE}"
   done
 }
+
+#######################################
+# Get a secret value from the key vault
+# Arguments:
+#   1: The name of the key vault
+#   2: The name of the secret (used to identify it e.g. "postgres-password")
+#######################################
+function azure::get_secret_value() {
+  az keyvault secret show --vault-name "${1}" --name "${2}" --query value -o tsv
+}
+
+#######################################
+# Assign the role 'Storage Blob Data Contributor' to the current user
+#######################################
+function azure::assign_storage_blob_data_contributor_role_to_user() {
+  local user_id=$(az ad signed-in-user show --query objectId -o tsv)
+  local subscription_id=$(az account show --query id -o tsv)
+  local role_assignments=$(az role assignment list --assignee ${user_id})
+  if [[ ("Storage Blob Data Contributor" == *$role_assignments*) ]];
```

Comment:
issue (blocking): this looks like it may file if the user has more role assignments than only "Storage Blob Data Contributor"

Reply:
🙈  I totally misread the `*`s as quotes despite syntax highlighting, my mistake. sorry!